jaxlib/python_extension.bzl

```diff
@@ -0,0 +1,61 @@
+# Copyright 2025 The JAX Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
+load("@symbol_locations//rules_pywrap:pywrap.bzl", "pybind_extension")
+
+def pywrap_extension(
```

Comment:
Can you please keep this wrapper still named as "pybind_extension"? Also, as we discussed, I'm planning to incorporate this logic into the rules themselves anyways (after this and TF's part land, as i don't want to retest this thing again on all platforms before it all settles down), and in that case the user-facing macro will be still called pybind_extension.

I'm just trying to keep terminology of these rule set consistent and well-defined across different repositories and google internal repositories. There have been so many different versons of the same thing with different prefixes/suffixes/completely_different_names across google repositories, so I'm really trying to avoid that happening again with new rules.

Reply:
Nope. This has nothing to do with pybind. We do not use pybind11 in this package. Your rule is not well named.